ament_cmake_pytest/cmake/ament_has_pytest.cmake

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Check if the Python module `pytest` was found.
+#
+# :param var: the output variable name
+# :type var: string
+# :param QUIET: suppress the CMake warning if pytest is not found, if not set and pytest was not found a CMake warning is printed
+# :type QUIET: option
+# :param PYTHON_EXECUTABLE: absolute path to the Python interpreter to be used,
+#   default to the CMake variable with the same name returned by
+#   FindPythonInterp
```

Comment:
Does this need to be wrapped ? seems more readable if it was on the line above

Reply:
It didn't fit within 80 characters.

Reply:
Ok, in that case, is it possible to wrap the parameter description just above (QUIET) that is much longer than this one ?

Reply:
Done in dcfedfbe5c43a47cb7f29a41132fd7bad81680ba.